site/content/en/docs/manual/basics/attach-cloud-storage.md

```diff
@@ -0,0 +1,63 @@
+---
+title: 'Attach cloud storage'
+linkTitle: 'Attach cloud storage'
+weight: 21
+description: 'Instructions on how to attach cloud storage using UI'
+---
+
+In CVAT you can use AWS-S3 and Azur Blob Container cloud storages to store image datasets for your tasks.
```

Comment:
Can we prepare a tutorial here? Let's try to create "AWS account", "load images to S3", "prepare the manifest", "load manifest to S3", "use the storage to create tasks on CVAT.

Reply:
Users should be able to reproduce our steps. For now we have only description for the complex process.

Reply:
It is also a good idea to get a pubic dataset for the purpose.